pkg/types/hashedrekord/v0.0.1/hashedrekord_v0_0_1_schema.json

```diff
@@ -38,7 +38,7 @@
                         "algorithm": {
                             "description": "The hashing function used to compute the hash value",
                             "type": "string",
-                            "enum": [ "sha256" ]
+                            "enum": [ "sha256", "sha512" ]
```

Comment:
This will be the first time we're adding a non-sha256 hash, so we'll want to double check and test we don't hardcode it elsewhere.